jaeseon/app.js

```diff
@@ -34,6 +34,20 @@ app.use(morgan('combined'));
 app.get('/ping', (req, res) => {
   return res.status(200).json({ message: 'pong' });
 });
+app.post('/users', async (req, res) => {
+  const { name, email, profile_image, password } = req.body;
+  await myDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      profile_image,
+      password
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [name, email, profile_image, password]
+  );
+  res.status(201).json({ message: 'userCreated' });
+});
```

Comment:
```suggestion
app.post('/users', async (req, res) => {
  const { name, email, profile_image, password } = req.body;
  await myDataSource.query(
    `INSERT INTO users(
      name,
      email,
      profile_image,
      password
    ) VALUES (?, ?, ?, ?);
    `,
    [name, email, profile_image, password]
  );
  return res.status(201).json({ message: 'userCreated' });
});
```
재선님-! 코드 잘 작성해주셨네요~ 👍 다만 아까 구두로 말씀 드린것처럼 migration 파일은 Westudy SQL part에서 `Assignment 1 | Database Schema & table 구축`을 참고하여 다시 작성해주세요 :) main에 merge된 기존 migration 파일들은 지워주셔도 됩니다!